src/OpenTelemetry/Resources/ResourceBuilderExtensions.cs

```diff
@@ -90,7 +90,7 @@ public static class ResourceBuilderExtensions
         /// <summary>
         /// Adds service information to a <see cref="ResourceBuilder"/>
         /// following <a
-        /// href="https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk">semantic
+        /// href="https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#telemetry-sdk">semantic
```

Comment:
this change is incorrect. the branch was named to "main" last week.

Reply:
I noticed this, but when I use .../master/... the link doesn't properly direct to the right part of the page. /main/ will show the warning but also direct to the right section of the page.
I'll still change it to master.